djedi/plugins/img.py

```diff
@@ -46,7 +75,6 @@ def load(self, content):
             return {'filename': None, 'url': None}
 
     def save(self, data):
-        from PIL import Image
```

Comment:
I don't remember why this import were done here, but maybe to not require PIL to be installed if you don't need the img plugin? Maybe keep the import here to not raise an ImportError?

Reply:
That's true, I didn't consider this one.